test/test_emotion_classifier.py

```diff
@@ -0,0 +1,55 @@
+# coding: utf-8
+
+from __future__ import absolute_import
+
+import urllib3
+import unittest
+
+import feersum_nlu
+from feersum_nlu.rest import ApiException
+from test import feersumnlu_host, feersum_nlu_auth_token
+
+
+class TestEmotion(unittest.TestCase):
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    def test_emotion(self):
+        # Configure API key authorization: APIKeyHeader
+        configuration = feersum_nlu.Configuration()
+
+        # configuration.api_key['AUTH_TOKEN'] = feersum_nlu_auth_token
+        configuration.api_key['X-Auth-Token'] = feersum_nlu_auth_token  # Alternative auth key header!
+
+        configuration.host = feersumnlu_host
+
+        api_instance = feersum_nlu.EmotionClassifiersApi(feersum_nlu.ApiClient(configuration))
+
+        model_instance_name = 'generic'
+        text_input = feersum_nlu.TextInput("I am very happy to hear that?")  # TextInput | The input text.
```

Comment:
If the model is ever retrained, we might want to test that the other labels are correctly identified. So maybe a `text_input` for the other labels ?

Reply:
True, this was more to test the interface, but one should really test more of the classes.